examples/visualisation/visualise_data.ipynb

```diff
@@ -332,7 +332,7 @@
     "cfg[\"raster_params\"][\"map_type\"] = \"py_semantic\"\n",
     "rast = build_rasterizer(cfg, dm)\n",
     "dataset = EgoDataset(cfg, zarr_dataset, rast)\n",
-    "scene_idx = 2\n",
```

Comment:
Any reason for this?
I see in the text though it says 'we visualise the second scene' - which is unchanged - was that a little error before (i.e. we start counting from 0) or does that need to be changed here?

Reply:
I've changed the text to reflect this. Scene 1 is more interesting :)